packages/create-rise/src/cli.ts

```diff
@@ -48,7 +58,10 @@ async function createRise() {
       },
     ])
     if (!overwrite) {
-      console.error(`The directory "${root}" already exists. Exiting...`)
+      console.error(dedent`
+        ${chalk.red.bold('🚨 There was an error setting up new Rise project:')}
+        ${chalk.red(`The directory "${root}" already exists. Exiting...`)}
+        `)
```

Comment:
nit: make sure to run ESLint on save. Line 64 looks like bad indentation which should've been fixed by ESLint already!

Reply:
Weird, it didn't pick it up as an error. My `vscode` should pick the `.vscode/settings.json` by default i think 🤔